api/src/pcapi/routes/backoffice/templates/venue/get.html

```diff
@@ -274,6 +274,18 @@ <h5 class="modal-title">Supprimer le lieu {{ venue.common_name }}</h5>
                   {{ venue.criteria | format_criteria | safe }}
                 </p>
               {% endif %}
+              {% if venue.accessibilityProvider %}
+                <p class="mb-1">
+                  <span class="fw-bold">Lien vers la page Acceslibre :</span>
+                  {% if venue.accessibilityProvider.externalAccessibilityUrl %}
+                    <a href="{{ venue.accessibilityProvider.externalAccessibilityUrl | format_website }}"
+                       target="_blank"
+                       class="link-primary">{{ venue.accessibilityProvider.externalAccessibilityUrl | format_website }}</a>
```

Comment:
Pour les liens externes, on met une petite icône à droite.</i>
```suggestion
                       class="link-primary">{{ venue.accessibilityProvider.externalAccessibilityUrl | format_website }} <i class="bi bi-box-arrow-up-right"></a>
```

Je pense aussi à la PR https://github.com/pass-culture/pass-culture-main/pull/11780 qui ajoute une vérification des liens externes, mais comme c'est une saisie d'une adresse vers un site a priori sûr, on peut sans doute s'en passer.